PSReadLine/ReadLine.cs

```diff
@@ -877,7 +861,7 @@ private void DelayedOneTimeInitialize()
             _readKeyWaitHandle = new AutoResetEvent(false);
             _keyReadWaitHandle = new AutoResetEvent(false);
             _closingWaitHandle = new ManualResetEvent(false);
-            _requestKeyWaitHandles = new WaitHandle[] {_keyReadWaitHandle, _closingWaitHandle, null};
```

Comment:
Ah ha, the questions I had earlier but hadn't written yet are now irrelevant. Yay!